src/cms-content/articles/index.ts

```diff
@@ -1,13 +1,17 @@
 import moment from 'moment';
 import { Markdown, sanitizeID } from '../utils';
-// import indigenousPeoplesDay from './indigenous-peoples-day.json';
+import { keyBy } from 'lodash';
+import indigenousPeoplesDay from './covid-impact-majority-native-american-counties.json';
+import thirdSurge from './third-surge.json';
+import animap from './animap.json';
+import canCompare from './can-compare.json';
 
 interface ArticleMain {
   articleID: string;
   header: string;
-  subtitle: string;
-  summary: Markdown;
-  author: Markdown;
+  subtitle?: string;
+  summary?: Markdown;
```

Comment:
Maybe we shouldn't make the subtitle optional, that could break the landing page for Articles

Reply:
the `summary` is the text shown on the landing page--good point, ill make it required. im not sure what the intended use of `subtitle` was though. maybe it can go?

Reply:
update: will remove the `subtitle` field (and maybe the `author` field?) once i get signoff on the layout. adding a todo